Makefile

```diff
@@ -0,0 +1,15 @@
+all: test
```

Comment:
That said, I'm highly in favor of adding Makefiles. @jbenet's been going on about them for a long time and I didn't pay attention until very recently and he's totally right about their value. They just make it so much easier to get started than decoding what's in a specific package.json file.